docs/application/web/guides/sensors/ham.md

```diff
@@ -310,6 +310,92 @@ Learning how to detect whether the user is asleep is a basic Human Activity Moni
    tizen.humanactivitymonitor.stop('SLEEP_MONITOR');
    ```
 
+## Detecting Sleep
+
+Learning how to detect whether the user is asleep is a basic Human Activity Monitor (HAM) management skill:
+
+1. To enable the monitor and start collecting data, use the `start()` method of the `HumanActivityMonitorManager` interface (in [mobile](../../../../org.tizen.web.apireference/html/device_api/mobile/tizen/humanactivitymonitor.html#HumanActivityMonitorManager) and [wearable](../../../../org.tizen.web.apireference/html/device_api/wearable/tizen/humanactivitymonitor.html#HumanActivityMonitorManager) applications) with
+`HumanActivityType` set to SLEEP_DETECTOR:
+
+   ```
+   function onchangedCB(sleepInfo) {
+       console.log('Sleep status: ' + sleepInfo.status);
+   }
+
+   tizen.humanactivitymonitor.start('SLEEP_DETECTOR', onchangedCB);
+   ```
+
+2. To disable the monitor when it is no longer required, use the `stop()` method of the `HumanActivityMonitorManager` interface with
+`HumanActivityType` set to SLEEP_DETECTOR:
+   ```
+   tizen.humanactivitymonitor.stop('SLEEP_DETECTOR');
+   ```
+
+## Monitoring Stress
+
+Learning how to register a listener that allows you to monitor user's stress is a basic Human Activity Monitor (HAM) management skill:
+
+1. To register an event handler for monitoring user stress, use the `addStressMonitorChangeListener()`  method of the `HumanActivityMonitorManager` interface (in [mobile](../../../../org.tizen.web.apireference/html/device_api/mobile/tizen/humanactivitymonitor.html#HumanActivityMonitorManager) and [wearable](../../../../org.tizen.web.apireference/html/device_api/wearable/tizen/humanactivitymonitor.html#HumanActivityMonitorManager) applications):
+
+   ```
+   var listenerId;
+
+   function errorCallback(error)
+   {
+       console.log(error.name + ": " + error.message);
+   }
+
+   function listener(label)
+   {
+       console.log("Stress level: " + label);
+   }
+
+   var ranges = [new tizen.StressMonitorDataRange("Normal", 10, 50),
```

Comment:
Normal state is 10 to 50, but alarms are 0-20 and 50-infinity and 30-100. Please change labels or reorganize ranges.